xc/device.ml

```diff
@@ -2489,6 +2485,36 @@ module Backend = struct
             ; xen_platform_device
             ] in
 
+        let disks' =
+          let format_of_media (media:Dm_Common.media) file =
+            match media, file with
+            | Dm_Common.Disk, _   -> ["format=raw"]
+            | Dm_Common.Cdrom, "" -> []
+            | Dm_Common.Cdrom, _  -> ["format=raw"]
+          in
+          let lba_of_media (media:Dm_Common.media) =
+            match media with
+            | Dm_Common.Disk  -> "force-lba=on"
+            | Dm_Common.Cdrom -> "force-lba=off"
+          in
+          List.map (fun (index, file, media) ->
+            let file = file ^ match String.startswith "nbd:unix:" file with
+              | true  -> ":exportname=qemu_node"
```

Comment:
This code will not be required after the rebase (see the latest `qemu-wrapper`), now we include the `:exportname=...` part in the file URI

Reply:
I've removed this chunk for the changes going to xenopsd/master in https://github.com/xapi-project/xenopsd/pull/527